src/MessagePack/Resolvers/StandardResolver.cs

```diff
@@ -25,6 +26,7 @@ public sealed class StandardResolver : IFormatterResolver
         /// </summary>
         public static readonly MessagePackSerializerOptions Options;
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL3050", Justification = Constants.AvoidDynamicCodeRuntimeCheck)]
         private static readonly IFormatterResolver[] Resolvers = DynamicAssembly.AvoidDynamicCode
```

Comment:
If it only silences analyzer warnings when targeting net9.0, that won't work since I'm also targeting net8.0 and need to suppress warnings there anyway.